scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -288,7 +288,7 @@
           <span class="float-end">{{ variant.spidex_human }}</span>
         </li>
         <li class="list-group-item">
-          <a href="https://doi.org/10.1016/j.cell.2018.12.015" target="_blank">SpliceAI</a> <a href="https://github.com/Illumina/SpliceAI" target="_blank">DS max</a>
+          <a href="https://doi.org/10.1016/j.cell.2018.12.015" target="_blank">SpliceAI</a> (<a href="{{ variant.spliceai_link }}">Lookup</a>) <a href="https://github.com/Illumina/SpliceAI" target="_blank">DS max</a>
```

Comment:
Nice! Perhaps a couple of links here are a bit redundant? If you open the github repo page with the third link it already points you to the paper, so perhaps remove the first link? 
 would also make the new link open in a new window. I'll add a suggestion!